src/containers/Quicklinks.js

```diff
@@ -71,19 +71,42 @@ export const QuickLinks = () => {
       })
       .then(links => {
         //TODO: Group links into categories
-        console.log(links)
-        setLinks(links)
+        const dummyLinks = [
```

Comment:
Yeah, this looks good! Don't forget to make a PR for this on https://github.com/nwplus/database-schemas/ to update the quick link schema.

After that's done, we can transition this to pull the data from Firebase

Reply:
Cool, good reminder, thanks!

Reply:
Ohp I think the schema was already good! I've removed the dummy links now that we know the schema will work with the code 👍 
![image](https://user-images.githubusercontent.com/47487758/97537541-2efe7800-197c-11eb-885f-6bb76bb31a60.png)